pkg/token/serviceclient/handler.go

```diff
@@ -149,6 +153,7 @@ func (h *Handler) retrieveToken() common.Result {
 		}
 
 		// If token is about to expire in TimeToTokenExpiry seconds or less generate a new one
+		spew.Dump(tokenDetails)
```

Comment:
Is this just a fancy print statement? Do we want to print out the token that may still be valid for `TimeToTokenExpiry` seconds?